### PR TITLE
Du/bg 189/handle routes

### PR DESCRIPTION
**Pull request by Don-PhDev**
After merging I will apply it to all controller

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -6,4 +6,8 @@ class ApplicationController < ActionController::Base
   def after_sign_out_path_for(_resource_or_scope)
     new_user_session_path
   end
+
+  def routing_error
+  render file: "#{Rails.root}/public/404.html", status: 404
```
Comment (rusbal) [inline app/controllers/application_controller.rb]: Improve code alignment.

File: app/controllers/transactions_controller.rb [hunk 2]
```diff
@@ -81,6 +83,10 @@ def transaction_params
 
   def set_transaction
     @transaction = Transaction.find(params[:id])
+
+  rescue ActiveRecord::RecordNotFound
+    redirect_to transactions_path,
+      alert: "Transaction #{transaction_url.split("/").last} was deleted or might never existed"
```
Comment (rusbal) [inline app/controllers/transactions_controller.rb]: Simply call `routing_error`

File: app/policies/entity_policy.rb [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  def create?
+    user.admin?
+  end
+
+  def destroy?
+    user.admin?
+  end
+
+  def see?
+    user.admin?
```
Comment (rusbal) [inline app/policies/entity_policy.rb:11]: Everybody should be able to see.  Return `true`.

**Review by rusbal — CHANGES_REQUESTED** [review]
See my comments.

File: app/policies/entity_policy.rb (hunk 3 above)
Comment (Don-PhDev) [reply app/policies/entity_policy.rb:11]: Screen Shot 2021-10-05 at 1 45 11 PM
Screen Shot 2021-10-05 at 1 45 01 PM

Wala na po kasing natitirang action na pwedeng gawin si ordinary user(Don) kaya nilagyan ko niyan para hindi niya makita `Actions`

File: app/controllers/transactions_controller.rb (hunk 2 above)
Comment (Don-PhDev) [reply app/controllers/transactions_controller.rb]: Parang magkaiba po kasi sila e

https://user-images.githubusercontent.com/10540193/135968966-08d1b613-8f23-41bd-94a4-c025f513b43b.mov

Comment (rusbal) [reply app/controllers/transactions_controller.rb]: Not found is 404.  Doesn't matter if the user is deleting, editing, just changing the id to a high number.

**Pull request merged**